src/view/ReferralsView.vue

```diff
@@ -1,10 +1,136 @@
+                </transition>
+            </teleport>
+        </div>
+        <article class="referrals__listOf"></article>
+        <article class="referrals__import"></article>
+        <aside class="referrals__userInfo">
+            <user-sidebar-component />
+        </aside>
+    </section>
 </template>
 
-<script setup></script>
+<script setup>
+    import ButtonComponent from '../components/ButtonComponent.vue';
+    import useUserStore from '../stores/userStore.js';
+    import ModalComponent from '../components/ModalComponent.vue';
+    import GreetingsComponent from '../components/GreetingsComponent.vue';
+    import UserSidebarComponent from '../components/profile/userSidebar/UserSidebarComponent.vue';
+    import ImportReferralsComponent from '../components/referrals/ImportReferralsComponent.vue';
+    import ReferralsComponent from '../components/referrals/ReferralsComponent.vue';
+
+    import { onMounted, ref } from 'vue';
+    import { onClickOutside } from '@vueuse/core';
+
+    const userStore = useUserStore();
+    const openModalReferrals = ref(false);
+    const openModalImportReferrals = ref(false);
+    const modalReferrals = ref(null);
+    const modalImportReferrals = ref(null);
+
+    onClickOutside(modalReferrals, () => (openModalReferrals.value = false));
+
+    onClickOutside(
+        modalImportReferrals,
+        () => (openModalImportReferrals.value = false)
+    );
 
+    onMounted(async () => {
+        await userStore.fetchUserData();
+        await userStore.fetchPatientReferrals();
+        await userStore.fetchPatientImportReferrals();
+    });
+</script>
 <style scoped lang="scss">
-    * {
+    .referrals {
+        display: grid;
+        place-items: flex-start;
+        height: 100vh;
+        width: 100%;
+        gap: 2%;
+        margin-inline: 2rem;
+        grid-template-areas:
+            'dashboard dashboard dashboard dashboard dashboard dashboard dashboard sidebar'
+            'actions actions actions actions actions actions actions sidebar'
+            'listof listof listof listof listof listof listof sidebar'
+            'listof listof listof listof listof listof listof sidebar'
+            'import import import import import import import sidebar'
+            'import import import import import import import sidebar';
+        &__header {
+            height: 50%;
+            grid-area: dashboard;
+            @include text-header1($font-weight-semiBold);
+            color: $blue-900;
+            place-self: center flex-start;
+        }
+        &__actions {
+            grid-area: actions;
+            place-self: space-between;
+            width: 50%;
+            gap: 2rem;
+            @include flex-position(row, nowrap, flex-start, center);
+        }
+        &__listOf {
+            grid-area: listof;
+            place-self: flex-start;
+        }
+        &__import {
+            grid-area: import;
+            place-self: flex-start;
+        }
+        &__userInfo {
```

Comment:
why you adding style about colors etc about component that was created once?

Reply:
i will use it later, that why is created now.

Reply:
No you didn't understand me. But forget about it. I have to refactor few style for usersidebar and move info about colors and border radius to exact component not above him.